ui/component/commentMenuList/view.jsx

```diff
@@ -171,9 +179,51 @@ function CommentMenuList(props: Props) {
       .then(() => doToast({ message: __('Link copied.') }));
   }
 
+  function reduceUriToChannelName(uri: string) {
+    try {
+      return uri.substring(uri.indexOf('@'), uri.indexOf('#') + 2).replace('#', ':');
+    } catch {
+      return uri;
+    }
+  }
```

Comment:
Just taking 1 character from the hash will not work for some cases.

If both `@thomas:b` and `@thomas:bf` happened to commented, clicking 'reply' on the latter's comment ends up pre-filling the chat with `@thomas:b`, which ends up replying to the wrong person.

Not sure there the disambiguated form comes from ... I think it's from the resolved canonical url (Haffa/Tom would know).

<img width="177" alt="image" src="https://user-images.githubusercontent.com/64950861/180137215-866fd116-0b0a-44d4-87c0-f6e727197822.png">

Reply:
I've copied this part from the blockList component. I guess I'll have to use parseURI instead.

Reply:
I think parseURI wouldn't help either ... I think 1 character vs 2 character hash is related to who has the highest bid?
Tom explained to me before, but I still never understood it, lol.

Reply:
probably need to get from resolved data
<img width="390" alt="image" src="https://user-images.githubusercontent.com/64950861/180149239-498df8a4-0eba-4ba3-9079-2e0abb7147a0.png">

Reply:
Yeah, that's already fixed. I noticed it right after I wrote that.